src/coll/error.rs

```diff
@@ -114,55 +104,44 @@ impl fmt::Display for BulkWriteException {
 
 impl fmt::Display for BulkWriteError {
     fn fmt(&self, fmt: &mut fmt::Formatter) -> fmt::Result {
-        try!(write!(
+        write!(
             fmt,
             "BulkWriteError at index {} (code {}): {}\n",
             self.index,
             self.code,
             self.message
-        ));
+        )?;
 
         match self.request {
-            Some(ref request) => try!(write!(fmt, "Failed to execute request {:?}\n.", request)),
-            None => {
-                try!(write!(
-                    fmt,
-                    "No additional error information was received.\n"
-                ))
-            }
+            Some(ref request) => write!(fmt, "Failed to execute request {:?}\n.", request),
```

Comment:
Oh, sure, that's a tpyo. Also I don't understand the newline part: do we want to include the newline and just change it to `writeln`, or do we not want the newline at all? (I didn't remove the newlines because I specifically didn't want to change the output.)

Reply:
I don't think we want the strings to end in newlines; I'd be okay with changing the output for error messages to not include it, but also happy to keep it as is.